nibabel/tests/test_fileutils.py

```diff
@@ -0,0 +1,57 @@
+# emacs: -*- mode: python-mode; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+# ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#
+#   See COPYING file distributed along with the NiBabel package for the
+#   copyright and license terms.
+#
+# ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+""" Testing fileutils module
+"""
+
+import numpy as np
+
+from ..fileutils import read_zt_byte_strings
+
+from numpy.testing import (assert_almost_equal,
+                           assert_array_equal)
+
+from nose.tools import (assert_true, assert_false, assert_raises,
+                        assert_equal, assert_not_equal)
+
+
+from ..tmpdirs import InTemporaryDirectory
+
+
+def test_read_zt_byte_strings():
+    # sample binary block
+    binary = b'test.fmr\x00test.prt\x00something'
+    with InTemporaryDirectory():
+        # create a tempfile
+        path = 'test.bin'
+        fwrite = open(path, 'wb')
+        # write the binary block to it
+        fwrite.write(binary)
+        fwrite.close()
+        # open it again
+        fread = open(path, 'rb')
+        # test readout of one string
+        assert_equal([s for s in read_zt_byte_strings(fread)],
```

Comment:
This should be possible without a list comprehension as the return value is no generator anymore, right?